tests/system/libraries/SystemTestSpy/speechSpyGlobalPlugin.py

```diff
@@ -275,14 +282,25 @@ def dump_speech_to_log(self):
 			except Exception:
 				log.error("Unable to log speech")
 
-	def dump_braille_to_log(self):
+	def _dump_braille_to_log(self):
+		"""Should only be called on main thread"""

```

Comment:
This docstring fails to explain why these methods have to be called from the main thread. Documenting the reasons may allow someone to lift this limitation in the future if this ever becomes a problem.